src/pages/admin/who-we-are/components/sections-wrapper/SectionsWrapper.scss

```
@@ -0,0 +1,47 @@
+        flex-direction: row;
+        align-items: center;
+        justify-content: flex-end;
+        gap: 0.625rem;
+        width: 100%;
+
+        .who-we-are-translate-btn {
+            border: none;
+            width: 1.5rem;
+            height: 1.5rem;
+
+            background: url('~@/assets/icons/translate.svg');
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)